### ISSUE TITLE
Runtime exception on composer install.

### ISSUE THREAD

**Issue by DiDebru** [reactphp-legacy/gifsocket#3]
[RuntimeException]                                                          
  Failed to execute git checkout 'v0.2.1' -- && git reset --hard 'v0.2.1' --  
  fatal: invalid reference: v0.2.1

**Comment by jsor** [comment]
Yes, composer is trying to install `react/event-loop:v0.2.1` which simply doesn't exist. Not sure why, probably due to the split of `react/react` into single components.

A quick fix is to remove `composer.lock` and just do a fresh `composer install`.

**Comment by WyriHaximus** [comment]
Actually this repo's tags also seem missing :zipper_mouth_face:

**Comment by jsor** [comment]
I'm not sure this repo ever had any tags ;)

**Comment by WyriHaximus** [comment]
Fair point :+1:
